tests/administration/realms.test.js

```diff
@@ -1,7 +1,54 @@
+'use strict';
 const common = require('../common'),
       expect = require('chai').expect;
 
 describe('Realms', () => {
   before(() => common.login(browser, 'admin', 'admin', 'local'));
+  after(() => common.logout());
   it('should navigate to Realms', () => common.clickSidebarTab(browser, 'Realms'));
+
+  it('should create a valid realm', () => {
+  	return common.waitForOnClick('//*[@id="add_realm"]')
+  		.then(() => browser.click('//*[@id="add_realm"]'))
+  		.then(() => browser.waitUntil( browser.getAttribute('//*[@id="realm"]','onchange')
+  			.then((onc) => onc !== null )
+  		))
+  		.then(() => browser.setValue('//*[@id="realm"]', 'test'))
+  		.then(() => browser.waitUntil( browser.getAttribute('//*[@id="fqdn"]','onchange')
```

Comment:
oh I see sorry this is for the `waitUntil`, maybe just clean the style a little bit:

```
  .then(() => browser.waitUntil(browser.getAttribute('//*[@id="fqdn"]','onchange')
    .then((onc) => onc !== null)));
```
?